cookbooks/bcpc-hadoop/templates/default/hdp_core-site.xml.erb

```diff
@@ -76,4 +79,41 @@
     <value>com.hadoop.compression.lzo.LzoCodec</value>
   </property>
 
+  <% if node[:bcpc][:hadoop][:enable_kerberos] == true then %>
+  <property>
+    <name>hadoop.security.authentication</name>
+    <value>kerberos</value>
+    <description>Set theauthentication for the cluster. Valid values are: simple or kerberos.</description>
+  </property>
+
+  <property>
+    <name>hadoop.security.authorization</name>
+    <value>true</value>
+    <description>Enable authorization for different protocols.</description>
+  </property>
+
+  <property>
+    <name>hadoop.security.auth_to_local</name>
+    <value>
+      <% node[:bcpc][:hadoop][:kerberos][:data].each do|ke,va| %>
+      RULE:[2:$1@$0](<%= va['principal']%>@.*<%= node[:bcpc][:hadoop][:realm] %>)s/.*/<%=va['owner'] %>/
```

Comment:
Should these be attributes or some such. For sites that want to protect cluster one from cluster two `.*<realm>` might be a bit fast and loose?